### PR TITLE
ZIOS-10799: Moved fetch of quotes to Data Model

### PR DESCRIPTION
**Pull request by nicorsm**
## What's new in this PR?
 
Moved code from https://github.com/wireapp/wire-ios/pull/2899 inside the `ZMConversation` extension available in `DraftMessage`.

### CODE REVIEW

File: Source/Public/DraftMessage.swift [hunk 1]
```diff
@@ -132,6 +132,12 @@ fileprivate struct StorableQuote: Codable {
         }
 
     }
+    
+    public func fetchQuoteFromDraftMessage() -> ZMMessage? {
```
Comment (typfel) [inline Source/Public/DraftMessage.swift]: Why is this necessary? Isn't the message is fetched in: 
`func quote(in context: NSManagedObjectContext) -> ZMMessage?`

Comment (nicorsm) [reply Source/Public/DraftMessage.swift]: Conversation is `nil` when fetched from there...

Comment (typfel) [reply Source/Public/DraftMessage.swift]: But that can be fixed.

Comment (typfel) [reply Source/Public/DraftMessage.swift]: You could for example add the conversation parameter to the `draftMessage(in: context)` and then use `ZMMessage.fetch(withNonce: nonce, for: self, in: context)` in `quote(in:)`

**Comment by nicorsm** [comment]
@zenkins release minor
